src/includes/grouping/in-app/apple.mdx

```diff
@@ -1,6 +1,55 @@
+
 The sentry-cocoa SDK always marks public frameworks such as  UIKitCore, CoreFoundation, GraphicsServices, and so forth as `not inApp`.
-All private frameworks and the main executable inside the application bundle are marked as `inApp`. The following marks
-frames from libdispatch starting with `_dispatch_` as `inApp`.
+Before version 7.0.0, the sentry-cocoa SDK marks all private frameworks and the main executable inside the application bundle as `inApp`. Since version 7.0.0,
```

Comment:
I moved most of that text from the file below up here. So a full pass is not needed. To save you awesome reviewers some time :)